### ISSUE TITLE
Double wrapped optionals using dot notation not formatted correctly

### ISSUE THREAD

**Issue by MrAdamBoyd** [nicklockwood/SwiftFormat#9]
I noticed that SwiftFormat doesn't format double wrapped optionals that use dot notation correctly. It takes the working code and changes it so it can't compile. Here's an example of working code:

```
let hello: String?? = "Hello"
let capitalHello = hello??.capitalizedString
```
After SwiftFormat:
```
let hello: String?? = "Hello"
let capitalHello = hello ??. capitalizedString
```

Notice the extra space before and after `??.` in the second line. This is reproducible 100% of the time for me.

**Comment by nicklockwood** [comment]
Thanks, I'll take a look.

**Comment by nicklockwood** [comment]
Fixed in 0.5.1
